submit_cgap/scripts/submit_ontology.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+
+from dcicutils.common import APP_FOURFRONT
+from ..submission import submit_any_ingestion
+from ..utils import script_catch_errors
+
+
+EPILOG = __doc__
+
+
+def main(simulated_args_for_testing=None):
+    parser = argparse.ArgumentParser(  # noqa - PyCharm wrongly thinks the formatter_class is invalid
+        description="Submits a gene list",
```

Comment:
Lifted from Gene list submission

Reply:
Also fixed locally. To be uploaded shortly.